PathPlanning/ReedsSheppPath/reeds_shepp_path_planning.py

```diff
@@ -110,114 +83,193 @@ def left_straight_left(x, y, phi):
     if t >= 0.0:
         v = mod2pi(phi - t)
         if v >= 0.0:
-            return True, t, u, v
+            return True, [t, u, v], ['L', 'S', 'L']
 
-    return False, 0.0, 0.0, 0.0
+    return False, [], []
 
 
-def left_right_left(x, y, phi):
-    u1, t1 = polar(x - math.sin(phi), y - 1.0 + math.cos(phi))
+def left_straight_right(x, y, phi):
```

Comment:
Is it possible to update the doc to describe mathematics for each path type?
https://github.com/AtsushiSakai/PythonRobotics/blob/master/docs/modules/path_planning/reeds_shepp_path/reeds_shepp_path_main.rst

Reply:
Sure... will do it soon

Reply:
I have updated the documentation... Please take a look into it.

Reply:
Oh, great document!!!. Thank you!!

<img width="1096" alt="image" src="https://github.com/AtsushiSakai/PythonRobotics/assets/3813847/e9a0f09f-02b2-4b08-9544-0dcc87d94608">